components/classes/aircraft-details.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/aircraft-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Aircraft Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": {
+    "industry": ["travel_and_hospitality"]
+  },
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Provides the ability to track aircraft details including tail number (ID) and type.",
```

Comment:
Let's make the description as short as possible- 
"aircraft details including tail number (ID) and type."